src/fragments/lib/datastore/react-native/getting-started/polyfills.mdx

```diff
@@ -0,0 +1,8 @@
+### Add Polyfills
+
+React Native is missing some standard library functionality that is required by DataStore.
```

Comment:
React Native is missing some JavaScript standard library functionality that DataStore depends on. Add the following to the entry point of your app to polyfill these missing functionalities: